src/app/features/container/container.tsx

```diff
@@ -97,10 +97,12 @@ export function Container() {
   const isLogoClickable = variant !== 'home' && !isRpcRoute(pathname);
   return (
     <>
-      <SwitchAccountDialog
-        isShowing={isShowingSwitchAccount}
-        onClose={() => setIsShowingSwitchAccount(false)}
-      />
+      {isShowingSwitchAccount && (
```

Comment:
There was a mistake here in that `<SwitchAccountDialog` shouldn't be in the DOM unless showing

Reply:
nice catch! I believe problem with requests not being cancelled on close in `SwitchAccountDialog` is here